Makefile

```
@@ -69,6 +89,16 @@ image: collector unittest
 		-t quay.io/stackrox-io/collector:$(COLLECTOR_TAG) \
 		$(COLLECTOR_BUILD_CONTEXT)
 
+ma-image: ma-collector unittest
+	make -C collector txt-files
+	docker buildx build --push --platform ${ARCH} \
+		--build-arg COLLECTOR_VERSION="$(COLLECTOR_TAG)" \
+		--build-arg MODULE_VERSION="$(MODULE_VERSION)" \
+		-f collector/container/Dockerfile \
+		-t quay.io/stackrox-io/collector:$(COLLECTOR_TAG)-slim \
+		-t quay.io/stackrox-io/collector:$(COLLECTOR_TAG)-base \
```

CI is failing because we are missing a tag without the suffixes
```suggestion
		-t quay.io/stackrox-io/collector:$(COLLECTOR_TAG)-slim \
		-t quay.io/stackrox-io/collector:$(COLLECTOR_TAG)-base \
		-t quay.io/stackrox-io/collector:$(COLLECTOR_TAG) \
```